.github/workflows/reusable-macos.yml

```diff
@@ -37,7 +37,9 @@ jobs:
         path: config.cache
         key: ${{ github.job }}-${{ inputs.os }}-${{ env.IMAGE_VERSION }}-${{ inputs.config_hash }}
     - name: Install Homebrew dependencies
-      run: brew install pkg-config openssl@3.0 xz gdbm tcl-tk make
+      run: |
+        brew install pkg-config openssl@3.0 xz gdbm tcl-tk@8 make
+        brew link tcl-tk@8
```

Comment:
I don't know brew: what is this new `brew link tcl-tk@8` command? `brew install` is not enough? Maybe add a comment to explain it?

Reply:
Because `tcl-tk@8` is an alternative version and not the latest, it installs as so-called ["keg-only"](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology), which means it doesn't get symlinked into the usual paths:

```console
❯ brew info tcl-tk@8
==> tcl-tk@8: stable 8.6.15 (bottled) [keg-only]
Tool Command Language
https://www.tcl-lang.org
Installed
/opt/homebrew/Cellar/tcl-tk@8/8.6.15 (3,069 files, 35.7MB)
  Poured from bottle using the formulae.brew.sh API on 2024-11-28 at 14:38:32
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/t/tcl-tk@8.rb
License: TCL
==> Dependencies
Required: openssl@3 ✔
==> Caveats
The sqlite3_analyzer binary is in the `sqlite-analyzer` formula.

tcl-tk@8 is keg-only, which means it was not symlinked into /opt/homebrew,
because this is an alternate version of another formula.

If you need to have tcl-tk@8 first in your PATH, run:
  echo 'export PATH="/opt/homebrew/opt/tcl-tk@8/bin:$PATH"' >> ~/.zshrc

For compilers to find tcl-tk@8 you may need to set:
  export LDFLAGS="-L/opt/homebrew/opt/tcl-tk@8/lib"
  export CPPFLAGS="-I/opt/homebrew/opt/tcl-tk@8/include"

For pkg-config to find tcl-tk@8 you may need to set:
  export PKG_CONFIG_PATH="/opt/homebrew/opt/tcl-tk@8/lib/pkgconfig"
==> Analytics
install: 12,709 (30 days), 12,737 (90 days), 12,738 (365 days)
install-on-request: 280 (30 days), 310 (90 days), 310 (365 days)
build-error: 8 (30 days)
```

Perhaps a comment like?
```yml
        brew install pkg-config openssl@3.0 xz gdbm tcl-tk@8 make
        # Because alternate versions are not symlinked into place by default:
        brew link tcl-tk@8
```